src/ui/hexedit.cc

```diff
@@ -222,6 +221,125 @@ HexEdit::HexEdit(FileBlobModel *dataModel, QItemSelectionModel *selectionModel,
   cursor_timer_.setInterval(700);
   cursor_timer_.start();
   connect(&cursor_timer_, &QTimer::timeout, this, &HexEdit::flipCursorVisibility);
+
+  createAction(util::settings::shortcuts::HEX_MOVE_TO_NEXT_CHAR, [this] () {
+    setSelection(current_position_ + 1, selection_size_);
+    if (!isByteVisible(current_position_)) {
+      scrollRows(1);
```

Comment:
This seems to apply to almost all implementations of the actions in this file, please fix all of them.